app/models/form/sales/questions/buyers_organisations.rb

```diff
@@ -27,9 +24,5 @@ def displayed_answer_options(_log, _user = nil)
     }
   end
 
-  def unanswered_error_message
```

Comment:
This is not actually used - it is perfectly fine to leave all these options blank.

Reply:
Ah, okay. I see!